lib/pages/movie_info_screen/genres_list_widget.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:the_movie_db_module2_part1/repository/genre_repository.dart';
+import '../../constants.dart';
+
+class GenresList extends StatelessWidget {
+  static const double containerBorderRadius = 15;
+  static const double containerPadding = 5;
+  static const double containerMargin = 6;
+  static const double itemsFontSize = 15;
+  final List<num>? genresIds;
```

Comment:
I don't think this list could be null at all. Worst case scenario it'll be empty

Reply:
Also, you're marking here as null and you then force-unwrap this on line 24, so dangerous!